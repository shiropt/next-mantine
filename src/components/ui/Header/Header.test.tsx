import { render, screen } from "@testing-library/react";
import { Header } from "./";

describe("Headerコンポーネントのテスト", () => {
  test("", () => {
    render(<Header />);
  });
});
