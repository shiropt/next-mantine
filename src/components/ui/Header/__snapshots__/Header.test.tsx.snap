// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderコンポーネントSnapShotテスト  1`] = `
<DocumentFragment>
  <nav
    class="mantine-Header-root mantine-r0ho0p"
    data-testid="email"
  />
</DocumentFragment>
`;

exports[`Headerコンポーネントのテスト  1`] = `
<DocumentFragment>
  <nav
    class="mantine-Header-root mantine-r0ho0p"
    data-testid="email"
  />
</DocumentFragment>
`;
